hasher-matcher-actioner/hmalib/lambdas/api/bank.py

```diff
@@ -19,14 +30,65 @@ def to_json(self) -> t.Dict:
         return {"banks": [bank.to_json() for bank in self.banks]}
 
 
-def get_bank_api(bank_table: Table) -> bottle.Bottle:
+@dataclass
+class BankMembersPage(JSONifiable):
+    bank_members: t.List[BankMember]
+
+    # deserializes to dynamo's exclusive_start_key. Is a dict
+    continuation_token: t.Optional[str]
+
+    def to_json(self) -> t.Dict:
+        result = asdict(self)
+        result.update(bank_members=[member.to_json() for member in self.bank_members])
+        return result
+
+
+def get_bucket_and_key(s3_url: str) -> t.Tuple[str, str]:
+    """TODO: Find a better place to park this function.
+
+    Given an s3 url like s3://asdfasd/asdfasdfadf, extracts bucket name and key
+    """
+    o = parse.urlparse(s3_url)
```

Comment:
super nit: URI/URN not URL, only think the change/clarification is worth it here because something like `https://s3.amazonaws.com/<bucket-name>.s3.amazonaws.com/images/<sub-key>?AWSAccessKeyId=<redacted>` won't actually work in this function.